test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-2.1
+2.2
```

Comment:
https://github.com/kubernetes/kubernetes/pull/79142 is specifically bumping this version as well; coordination needs to happen to ensure the versions/merges happen in the appropriate order.

Reply:
Correct. But if I'm not mistaken, a merge conflict will occur once one of the PR merges anyways.

Reply:
I'm going to assume we'll want to bump this to 2.3, and am fine aiming for all of the other agnhost PR's before going after the windows stuff